tests/bcl-test/templates/common/TestRunner.xUnit/XUnitTestRunner.cs

```diff
@@ -828,7 +830,13 @@ public override void WriteResultsToFile (TextWriter writer, Jargon jargon)
 						writer.WriteLine ($"{e}");
 					}
 					break;
-
+				case Jargon.NUnitV3:
+					try {
+						Transform_Results ("NUnit3Xml.xslt", assembliesElement, xmlWriter);
+					} catch (Exception e) {
+						writer.WriteLine ($"{e}");
```

Comment:
not sure if the compiler is smart enough to remove the `String.Format` call but it's easier on the eye to just
```csharp
writer.WriteLine (e);
```